### ISSUE TITLE
Provide Java SDK 1.0.7 at maven central

### ISSUE THREAD

**Issue by jbman** [ory/hydra#1590]
Please update and release latest version 1.0.7 of the Java SDK.

The maven pom file sdk/java/hydra-client-resttemplate/pom.xml still has version 1.0.0 set.
In maven central there is also only version 1.0.0 available: https://mvnrepository.com/artifact/com.github.ory/hydra-client-resttemplate

Version 1.0.0  has problems in in `AdminApi.getLoginRequest` and `AdminApi.getConsentRequest`: The URLs are built wrong with challenge as path variable instead of query parameter.
In addition AcceptLoginRequest.context is not available in 1.0.0.

TLDR: 1.0.0 gives a bad developer experience and 1.0.7 might fix those but is missing in maven central.

**Comment by aeneasr** [comment]
It appears as the SDK build pipeline is not properly invoked when releasing a new version. I have to see why that's the case.

**Comment by UkonnRa** [comment]
So any update? Is the latest hydra java client available? I cannot wait to use the latest version! Thanks for your efforts!

**Comment by mkzaker** [comment]
Any update?
Please i need it

**Comment by UkonnRa** [comment]
I found that all the clients(java, go and js) are just simply generated by swagger-codegen, without any special tuning, so I think users can just use swagger-codegen to generate their own client. Most of the generated clients from swagger need more tuning, after all...

**Comment by foresx** [comment]
I found that all java sdk version 1.0.0 is not fit with rest api's documentation.

**Comment by aeneasr** [comment]
Sorry for the frustration here - the main point being that we have no Java developers in our team and therefore very little first-hand knowledge of how the Java ecosystem works. We'll try to address this issue however in the next sprint!

**Comment by aeneasr** [comment]
Has anyone here a good guide to publishing packages at maven (central)?

**Comment by aeneasr** [comment]
Could someone in this issue confirm that:

```
<dependency>
  <groupId>sh.ory.hydra</groupId>
  <artifactId>hydra-client</artifactId>
  <version>0.0.0-alpha.17</version>
</dependency>
```

is working as expected? It's the latest SDK version but released under this alpha flag as I've been trying to get this release pipeline to work for several hours.

**Comment by aeneasr** [comment]
I'm closing this because I believe this is fixed.

**Comment by foresx** [comment]
> I'm closing this because I believe this is fixed.

Thanks a lot, sadly due to deadline I make my own login request to hydra, hoping I can use it next time.
